website/docs/docs/dbt-cloud/dbt-cloud-enterprise/audit-log.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Using the Audit Log for dbt Cloud Enterprise"
+id: audit-log
+description: "You can review actions in your organization, which can help troubleshoot issues and secure organization."
+sidebar_label: "Using the Audit log"
+---
+
+dbt Cloud’s Audit Log allows you to quickly review the actions performed by members of your organization. It includes details such as who performed the action, what the action was, and when it was performed. These details can be used in various troubleshooting and security scenarios.
+
+## Accessing the Audit Log
+
+To access Audit Log, click **Account Settings** and **Audit Log**
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-menu.png" title="Click Audit log in the Account settings menu."/>
```

Comment:
@JKarlavige Is there any way to edit the size of this images inline?

Reply:
It's not possible to add inline styles directly to the Lightbox component currently, as it doesn't accept a prop for inline styles. 

However, one way to accomplish this is to wrap a Lightbox component with a div, and add the inline styles there. An example of that would be:

```
<div style={{maxWidth: '200px'}}>
  <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/audit-log-menu.png" title="Click Audit log in the Account settings menu."/>
</div>
```

Reply:
Thank you Jason! I added these for all of the images.